build/dependenciesTest.props

```diff
@@ -2,14 +2,17 @@
   <PropertyGroup>
     <BannedApiAnalyzersVersion>3.3.4</BannedApiAnalyzersVersion>
     <DotNetCoreAppRuntimeVersion>2.1.30</DotNetCoreAppRuntimeVersion>
-    <MicrosoftAzureKeyVaultCryptographyVersion>2.0.5</MicrosoftAzureKeyVaultCryptographyVersion>
+    <MicrosoftAzureKeyVaultCryptographyVersion>3.0.5</MicrosoftAzureKeyVaultCryptographyVersion>
     <MicrosoftNETTestSdkVersion>17.11.0</MicrosoftNETTestSdkVersion>
+    <MicrosoftRestClientRuntime>2.3.24</MicrosoftRestClientRuntime>
     <NetStandardVersion>2.0.3</NetStandardVersion>
     <NewtonsoftVersion>13.0.3</NewtonsoftVersion>
+    <SystemNetHttp>4.3.4</SystemNetHttp>
     <SystemSecurityClaimsVersion>4.3.0</SystemSecurityClaimsVersion>
     <SystemTextJson>8.0.4</SystemTextJson>
+    <SystemTextRegularExpressions>4.3.1</SystemTextRegularExpressions>
     <XunitRunnerConsoleVersion>2.9.0</XunitRunnerConsoleVersion>
-    <XunitRunnerVisualStudioVersion>2.8.2</XunitRunnerVisualStudioVersion>
+    <XunitRunnerVisualStudioVersion>3.0.0-pre.30</XunitRunnerVisualStudioVersion>
```

Comment:
```suggestion    <XunitRunnerVisualStudioVersion>2.8.2</XunitRunnerVisualStudioVersion>
```

Reply:
@jmprieur not available on net462. [Id web](https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/Directory.Build.props#L19) is on a lower version though, so I don't think this is the root cause of the issue, but could be wrong.